test/integration/server-test.ts

```diff
@@ -6,6 +6,10 @@ import { promisify } from "util";
 import { Webhooks } from "../../src";
 import pushEventPayload from "../fixtures/push-payload.json";
 
+const signatureSha1 = "sha1=f4d795e69b5d03c139cc6ea991ad3e5762d13e2f";
+const signatureSha256 =
+  "sha256=4864d2759938a15468b5df9ade20bf161da9b4f737ea61794142f3484236bda3";
```

Comment:
tests failing. I hope is tha sha256 I generated wrong. Tomorrow I'll keep researching

Reply:
try 

```suggestion
  "sha256=2b49327af77d51c4c23700118b11d15c81d90c3bd57dafbbe32eb50085ce67e0";
```

That made the tests pass for me. I calculated it like this:

```js
sign(
  { secret: "mysecret", algorithm: "sha256" },
  pushEventPayload
);
```

You probably had a typo in the `secret`?

Reply:
Something wrong with the secret or payload I was passing yep.